product_docs/docs/tpa/23/rel_notes/tpa_23.28_rel_notes.mdx

```diff
@@ -13,7 +13,7 @@ New features, enhancements, bug fixes, and other changes in Trusted Postgres Arc
 | Enhancement | Added a new hook `barman-pre-config`. This hook is invoked after Barman is installed and its user is set up but before it is configured. It can be used for installing certificate files or other tasks which need the barman user to exist but which must be done before Barman is started. |
 | Enhancement | The key `elastic_ip` on an AWS instance in `config.yml` can be set to an elastic IP address that has already been allocated in order to assign it to this instance. |
 | Change | In Patroni clusters, TPA now sets up replicas before handing over control of the cluster to Patroni, rather than setting up the primary only and letting Patroni set up the replicas. |
-| Change | If harp_manager_user is defined in `config.yml`, TPA will create the user `harp_manager_user`, belonging to the `bdr_superuser` role, and set HARP manager to operate as this user instead of as the `postgres` superuser. |
+| Change | For new clusters, TPA will create the user specified by setting `harp_manager_user` (by default `harpmanager`), belonging to the `bdr_superuser` role, and set HARP Manager to operate as this user instead of the as the `postgres` superuser. For existing clusters, TPA will keep using `postgres` as the HARP Manager user, unless the user overrides this behavior by explicitly setting `harp_manager_user` in `config.yml`. |
```

Comment:
```suggestion
| Change | For new clusters, TPA will create the user specified by setting `harp_manager_user` (by default `harpmanager`), belonging to the `bdr_superuser` role, and set HARP Manager to operate as this user instead of `postgres` superuser. This does not affect the existing clusters where TPA will keep using `postgres` as the HARP Manager user, unless the user overrides this behavior by explicitly setting `harp_manager_user` to a different value in `config.yml`. |
```

Reply:
LGTM.
Making a couple of minor suggestions to create a little more clarity. OK to go without these as well.